src/index.html

```diff
@@ -1,12 +1,48 @@
 <!DOCTYPE html>
-<html lang="en">
+
+<html lang="en" class="page">
```

Comment:
It's not a good practice to add classes to html or body tags

Reply:
Hi @jv-aquino, I really appreciate your advice, I'm doing a code review today.

About this subject, this class in body and html tag was recommended in the HTML+CSS Advanced/BEM class in the video "Mix Block with parent element". Is it necessary to follow other rules? 

In this case, I need to change the solutions for the other tasks. Let me know if it really needed to be changed.

Reply:
Unnecessary class:

Just as @jv-aquino mentioned, there's no need to name html and body tags as they are unique.